AutocompleteClient/FW/Logic/Query/Builder/RequestBuilder.swift

```diff
@@ -13,6 +13,8 @@ class RequestBuilder {
     var queryItems = QueryItemCollection()
     var dateProvider: DateProvider
     
+    internal(set) var searchTerm = ""
```

Comment:
I would back out this change.  Search term is not globally used in communicating with our servers the way that the rest of the variables / functions in this file are.